modules/end-user-guide/partials/proc_building-custom-intellij-editor-images.adoc

```diff
@@ -0,0 +1,154 @@
+[id="building-custom-intellij-editor-images_{context}"]
+= Building custom IntelliJ editor images
+
+This procedure describes how to build an IntelliJ-based editor image using a custom archive and a custom image name.
+
+.Prerequisites
+
+* The `git`, `jq`, and `wget` tools are available.
+
+* Docker version 18.09 or greater, supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit].
+
+* Java Development Kit (JDK) version 11 is available.
```

Comment:
`available` seems both redundant and vague. Does JDK need to be installed or included in a container image - where does the user find it - in a workspace?